docker_tasks/vector_ingest/Dockerfile

```diff
@@ -1,19 +1,9 @@
-FROM --platform=linux/amd64 ghcr.io/lambgeo/lambda-gdal:3.5-python3.9
+FROM --platform=linux/amd64 python:3.9
```

Comment:
This was using a gdal specific lambda image. Will eventually have to replace with another gdal image.

Reply:
What `cmd` are you using that requires `gdal`?

Reply:
I see you are running `ogr2ogr`